dir2cast.php

```
@@ -945,6 +991,11 @@ public function init()
 	 */
 	protected function acquireLock()
 	{
+		if (!file_exists($this->temp_dir))
+		{
+			mkdir($this->temp_dir, 0755, true);
```

This should probably throw a new Exception if it fails.